ui/src/views/Settings.vue

```diff
@@ -187,8 +191,24 @@ export default {
         reports_international_prefix: "",
         timezone: "",
       },
+      warning: {
+        user_domain: "",
+      },
     };
   },
+  mounted() {
+    this.previousUserDomain = this.form.user_domain;
+    this.$refs.user_domain.$el.addEventListener(
```

Comment:
Vue support events like `@input` for this kind of things, see https://vuejs.org/guide/essentials/forms.html#form-input-bindings